{{cookiecutter.project_slug}}/backend/app/app/crud/crud_user.py

```diff
@@ -24,6 +24,11 @@ def create(self, db_session: Session, *, obj_in: UserCreate) -> User:
         db_session.refresh(db_obj)
         return db_obj
 
+    def update(self, db_session: Session, *, db_obj: ModelType, obj_in: UpdateSchemaType) -> ModelType:
+        if obj_in.password:
+            db_obj.hashed_password = get_password_hash(obj_in.password)
```

Comment:
I am not sure if this is the correct place to update the password.

Reply:
You probably want to use the `reset_password` method in [login.py](https://github.com/tiangolo/full-stack-fastapi-postgresql/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/backend/app/app/api/api_v1/endpoints/login.py)

Reply:
> You probably want to use the `reset_password` method in [login.py](https://github.com/tiangolo/full-stack-fastapi-postgresql/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/backend/app/app/api/api_v1/endpoints/login.py)

Shouldn't it be the other way around, e.g. the endpoint/login.py using CRUDUser?